tests/utils/int8_to_float_scale.cpp

```diff
@@ -28,29 +28,29 @@ int main(int argc, char **argv)
     }
 
     std::ifstream reader(argv[1], std::ios::binary);
-    _u32 npts_u32;
-    _u32 ndims_u32;
-    reader.read((char *)&npts_u32, sizeof(_s32));
-    reader.read((char *)&ndims_u32, sizeof(_s32));
+    uint32_t npts_u32;
```

Comment:
get_metadata()?

Reply:
Lets take this up separately https://github.com/microsoft/DiskANN/issues/294